packages/patternfly-3/patternfly-react-extensions/src/components/VerticalTabs/VerticalTabsTab.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Button, noop } from 'patternfly-react';
+
+const VerticalTabsTab = ({ className, title, active, onActivate, ...props }) => {
+  const classes = classNames('vertical-tabs-pf-tab', { active }, className);
```

Comment:
Is `{ active }` a classNames shorthand for `{'active': active === true}`

Reply:
yes